SourceX/storm/storm.cpp

```diff
@@ -703,7 +712,9 @@ BOOL SVidPlayContinue(void)
 		Uint32 format = SDL_GetWindowPixelFormat(window);
 		SDL_Surface *tmp = SDL_ConvertSurfaceFormat(SVidSurface, format, 0);
 #endif
-		ScaleOutputRect(&pal_surface_offset);
+#ifndef __AMIGA__
+		ScaleOutputRect(&pal_surface_offset); // somehow this lead to pb with amiga's SDL

```

Comment:
pb?

Reply:
hmm.. yeah I got strange result with that that call at one point in time. But with changes between lines 677-684 this is no more needed apparently.

Reply:
Well I'm just not sure what "pb" stands for in this case :)

Reply:
pb = problem
but I think we can remove the change because when I do so, the exe seem to work fine under uae.